config/_default/menus/menus.zh-cn.toml

```diff
@@ -56,18 +56,18 @@
     weight = 400
 [[footer]]
     parent = "column1"
-    name = "Documentation"
-    url = "/docs/"
+    name = "Download"
+    url = "/docs/install/"
```

Comment:
/download top level

But can have a link in docs/install or docs/download too.